src/citrine/_serialization/properties.py

```diff
@@ -379,6 +380,35 @@ def _serialize(self, value: tuple) -> list:
         return serialized
 
 
+class Enumeration(Property[BaseEnumeration, BaseEnumeration]):
```

Comment:
I think this should be `class Enumeration(Property[BaseEnumeration, String]):`, since the serialized type is a string

Reply:
Good catch; corrected to `str`